Presentation.Web/Controllers/API/V2/External/ItSystemUsages/Mapping/ItSystemUsageResponseMapper.cs

```diff
@@ -188,53 +188,24 @@ private static GeneralDataResponseDTO MapGeneral(ItSystemUsage systemUsage)
             };
         }
 
-        private RiskLevelChoice? MapRiskAssessment(ItSystemUsage systemUsage)
+        private static RiskLevelChoice? MapRiskAssessment(ItSystemUsage systemUsage)
         {
-            return systemUsage.preriskAssessment switch
-            {
-                RiskLevel.LOW => RiskLevelChoice.Low,
-                RiskLevel.MIDDLE => RiskLevelChoice.Middle,
-                RiskLevel.HIGH => RiskLevelChoice.High,
-                RiskLevel.UNDECIDED => RiskLevelChoice.Undecided,
-                null => null,
-                _ => throw new ArgumentOutOfRangeException()
-            };
+            return systemUsage.preriskAssessment?.ToRiskLevelChoice();
         }
 
         private static IEnumerable<TechnicalPrecautionChoice> MapPrecautions(ItSystemUsage systemUsage)
         {
-            if (systemUsage.precautionsOptionsAccessControl)
-                yield return TechnicalPrecautionChoice.AccessControl;
-            if (systemUsage.precautionsOptionsEncryption)
-                yield return TechnicalPrecautionChoice.Encryption;
-            if (systemUsage.precautionsOptionsLogning)
-                yield return TechnicalPrecautionChoice.Logging;
-            if (systemUsage.precautionsOptionsPseudonomisering)
-                yield return TechnicalPrecautionChoice.Pseudonymization;
+            return systemUsage.GetTechnicalPrecautions().Select(precaution => precaution.ToTechnicalPrecautionChoice());
         }
 
         private static DataSensitivityLevelChoice? MapDataSensitivity(ItSystemUsageSensitiveDataLevel arg)
         {
-            return arg.SensitivityDataLevel switch
-            {
-                SensitiveDataLevel.NONE => DataSensitivityLevelChoice.None,
-                SensitiveDataLevel.PERSONALDATA => DataSensitivityLevelChoice.PersonData,
-                SensitiveDataLevel.SENSITIVEDATA => DataSensitivityLevelChoice.SensitiveData,
-                SensitiveDataLevel.LEGALDATA => DataSensitivityLevelChoice.LegalData,
-                _ => null
-            };
+            return arg?.SensitivityDataLevel.Transform(x => x.ToDataSensitivityLevelChoice());
```

Comment:
Hvorfor Transform her i stedet for noget lign: "systemUsage.preriskAssessment?.ToRiskLevelChoice();" som ovenfor?